platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/merkle/VirtualRootNode.java

```diff
@@ -1249,13 +1250,15 @@ public long estimatedSize() {
      */
     @Override
     public void serialize(final SerializableDataOutputStream out, final Path outputDirectory) throws IOException {
-        final RecordAccessor<K, V> detachedRecords = pipeline.detachCopy(this, outputDirectory);
-        assert detachedRecords.getDataSource() == null : "No data source should be created.";
+        pipeline.pausePipelineAndRun("detach", () -> {
```

Comment:
Maybe we should introduce a constant?

Reply:
These labels aren't used anywhere, they are used only to set a temporary thread name. I'd probably drop them altogether rather than introduce a constant...